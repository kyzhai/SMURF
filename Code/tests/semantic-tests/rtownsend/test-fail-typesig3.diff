1c1
< Fatal error: exception Sast.Main_missing
---
> Fatal error: exception Sast.Type_mismatch("x")
