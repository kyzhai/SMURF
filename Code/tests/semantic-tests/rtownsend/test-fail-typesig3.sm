/* If a variable has been declared, it's type signature must match the type of the bound expression */

x = (2,3)$4

x :: Int

main = []
