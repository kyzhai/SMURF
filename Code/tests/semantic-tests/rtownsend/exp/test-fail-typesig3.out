Fatal error: exception Sast.Type_mismatch("x")
